itest.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
```

Comment:
This looks like a lot of duplication from https://github.com/github/gh-ost/blob/master/script/cibuild-gh-ost-replica-tests

Can we instead just call https://github.com/github/gh-ost/blob/master/script/cibuild-gh-ost-replica-tests? Or refactor into a more generic script? There's otherwise too much duplication; the entire test logic is rewritten here; I'd like to see both Docker and the GitHub CI use the same code.